src/components/trade/PriceImpact/utils.ts

```diff
@@ -73,7 +74,7 @@ function determinePriceWarning(params: PriceImpactArgs, impact: BigNumber | null
     !fillPrice ||
     !bestAskPrice ||
     // round to 5 places and check limit = fill
-    fillPrice.toFixed(12) === limitPrice.toFixed(12)
+    fillPrice.toFixed(lowerPrecision) === limitPrice.toFixed(lowerPrecision)
```

Comment:
Why is this?
what's the diference on `fillPrice.toFixed(lowerPrecision) === limitPrice.toFixed(lowerPrecision)` and case 4 and 3. 
Also, what's the difference in meaning of returnig an empty string and a `null`?

Reply:
hmm good eye. it's the same as cases are covered.